pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -335,16 +327,21 @@ func (l *listUsersQuery) expandTTU(
 		userObjectType, userObjectID := tuple.SplitObject(userObject)
 
 		pool.Go(func(ctx context.Context) error {
-			return l.expand(ctx, &openfgav1.ListUsersRequest{
-				StoreId:              req.GetStoreId(),
-				AuthorizationModelId: req.GetAuthorizationModelId(),
-				Object:               &openfgav1.Object{Type: userObjectType, Id: userObjectID},
-				Relation:             computedRelation,
-				UserFilters:          req.GetUserFilters(),
-				ContextualTuples:     req.GetContextualTuples(),
-			}, foundUsersChan)
+			newReq := clone(req)
+			newReq.Object = &openfgav1.Object{Type: userObjectType, Id: userObjectID}
+			newReq.Relation = computedRelation
+			return l.expand(ctx, newReq, foundUsersChan)
 		})
 	}
 
 	return pool.Wait()
 }
+
+func enteredLoop(req *internalListUsersRequest) bool {
```

Comment:
nitpick: perhaps `visitedPath` is more descriptive than enteredLoop?

Reply:
i feel the opposite, actually 🤔

Reply:
Only other thing here is "loop" vs "cycle". We only use loop to describe model cycles and even then they're seemingly synonymous. Can we land on `enteredCycle`?